libbeat/docs/highlights-7.0.0.asciidoc

```diff
@@ -0,0 +1,103 @@
+[[release-highlights-7.0.0]]
+=== 7.0.0 release highlights
+++++
+<titleabbrev>7.0.0</titleabbrev>
+++++
+
+Each release of {beats} brings new features and product improvements. 
+Here are the highlights of the new features and enhancements in 7.0.0.
+
+Refer to the {beats} <<breaking-changes-7.0, Breaking Changes>> and <<release-notes, 
+Release Notes>> for a list of bug fixes and other changes.
+
+//NOTE: The notable-highlights tagged regions are re-used in the
+//Installation and Upgrade Guide
+
+// tag::notable-highlights[]
+
+[float]
+==== Elastic Common Schema (ECS)
+
+The {ecs-ref}/index.html[Elastic Common Schema], or ECS, is an open source
+specification that defines a common set of document fields for event data
+ingested into {es}. ECS makes it dramatically easier for users to correlate data
+across sources and develop common content, such as dashboards and machine
+learning jobs.
+
+In 7.0, all {beats} and {beats} modules generate ECS format events by default.
+This means that adopting ECS is as easy as upgrading to {beats} 7.0. All {beats}
+module dashboards in 7.0 make use of ECS.
+
+Migrating to a common schema means that many fields have been renamed. We have
+developed an upgrade procedure that uses {es} field aliases to make the
+transition easier. After the upgrade is complete, we strongly advise that you
+adjust your custom {kib} dashboards, machine learning jobs, and other content to
+use the new ECS field names. 
+
+See the {beats-ref}/upgrading.html[{beats} upgrade documentation] for more
+information.
+
+[float]
+==== Index lifecycle management (ILM)
+
+In 6.6, {es} added advanced capabilities for index management. Rather than
+simply performing management actions on your indices on a set schedule, you can
+base actions on other factors such as shard size and performance requirements.
+You control how indices are handled as they age by attaching a lifecycle policy
+to the index template used to create them. You can update the policy to modify
+the lifecycle of both new and existing indices. This set of capabilities are
+grouped in the {ref}/index-lifecycle-management.html[index lifecycle management
+(ILM)] APIs.
+
+In 7.0, Beats default to rotate indices by using ILM policies, if the {es}
+version to which they connect supports ILM. The default policy rotates indices
+when they reach 50 GB or 30 days. You can edit the ILM policy by using the {kib}
+management UI, or directly via the {es} API.
+
+[float]
+==== Stack monitoring
+
+The full suite of modules to {stack-ov}/monitoring-production.html[monitor your
+{stack}] are now GA. These include the {metricbeat} modules for {es}, {ls}, and
+{kib}.
+
+In the future, we will switch to {metricbeat} as the recommended shipper
```

Comment:
Not sure how closely you want to follow the blog post. But I would write "recommended agent" instead.